### ISSUE TITLE
Log spam when trying to add tunl0 device

### ISSUE THREAD

**Issue by fasaxc** [projectcalico/felix#1008]
Seems we hit this retry pretty consistently when starting calico-node; then the retry succeeds:

```
016-04-25 11:59:03,224 [ERROR][76/3] calico.felix.frules 216: Failed to configure IPIP device, retrying...
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/calico/felix/frules.py", line 212, in install_global_rules
    _configure_ipip_device(config)
  File "/usr/lib/python2.7/site-packages/calico/felix/frules.py", line 322, in _configure_ipip_device
    "mode", "ipip"])
  File "/usr/lib/python2.7/site-packages/calico/felix/futils.py", line 340, in check_call
    args, retcode, stdout, stderr, input=input_str)
FailedSystemCall: Failed system call (retcode : 1, args : ('ip', 'tunnel', 'add', 'tunl0', 'mode', 'ipip'))
  stdout  : 
  stderr  : add tunnel "tunl0" failed: File exists

  input  : None
```

It'd be good to suppress the traceback for this expected case and downgrade to a warning.

**Comment by fasaxc** [comment]
May get to revisit when porting dataplane driver to golang.

**Comment by fasaxc** [comment]
Fixed in #1202
